### DISCUSSION TITLE
Can I add the discogs_album_ID field to my tags?

### DISCUSSION THREAD

**Discussion by SOY75** [beetbox/beets#3949] [Q&A] [answered]
Hi everyone
It's been a week that I'm trying to figure out how to do that, so I guess It's time to ask :)

I've already tagged all my collection with beets, after that, I've done some manual corrections and/or additions with MP3TAG that I'd like to save (like %Media% or I corrected a lot of genre tags).
I've seen that since last year, it is possible to get the discogs_release_id which is awesome, I would like to include it in my tags.

Would it be possible to do it, without touching any others tags which are already perfect to my eyes?
I've spent a lot of time reading the wiki/forum but I didn't find anything.
Ideally, I would like it to be added in a custom tag %Discogs_ID% but if It's easier, I could also put it in the comment that I never use for anything.

Thanks a lot

SOY

**Comment by sampsyo** [comment]
Hello! I'm not entirely sure what you mean... could you clarify what you mean by "adding" this tag? The `discogs_￼albumid` field is indeed set by the discogs plugin on import... but I gather that's not what you're trying to do?

**Comment by SOY75** [comment, marked as answer]
Hi sampsyo!
Sorry if it was unclear, my english is pretty poor.
I asked on IRC, and apparently, what I'm trying to do is not possible. :(

logs: https://i.imgur.com/TQhfv3U.png

Thanks for your reply, anyway :)
